compiler/main.bal

```diff
@@ -44,7 +44,8 @@ public function main(string[] filenames, *Options opts) returns error? {
     }
     nback:Options nbackOptions = {
         gcName: check nback:validGcName(opts.gc),
-        debugLevel: check nback:validDebugLevel(opts.debugLevel)
+        debugLevel: check nback:validDebugLevel(opts.debugLevel),
+        outDir: opts.outDir
```

Comment:
Why is this the right place to do this? Surely the locations in the BIR should have the right directory.

Reply:
locations in BIR have the path to file relative to where we run the compiler from but debug information needs it relative to outDir

Reply:
How is it useful for BIR to have locations relative to where we run the compiler?  The point of BIR is to create output. So the BIR locations should be relative to the output directory. Please propose appropriate design changes in the issue.

Reply:
Why is outDir still here?